api/test/unit/management/commands/test_backfillmoderationdecision.py

```diff
@@ -78,12 +80,13 @@ def test_create_moderation_decision_for_reports(
     assert f"Created 1 {media_type} moderation decisions from existing reports." in out
 
     decision = MediaDecision.objects.first()
-    assert decision.media_objs.count() == 1
     assert decision.action == expected_action
     assert decision.moderator.username == username
 
+    assert MediaDecisionThrough.objects.filter(decision=decision).count() == 1
+
     decision_through = MediaDecisionThrough.objects.first()
-    assert decision_through.media_obj == report.media_obj
+    assert str(decision_through.media_obj_id) == report.media_obj_id
```

Comment:
Aren't these statements equivalent? Why the change here? When would they not be equivalent? Django models essentially compare equality through `self.pk == other.pk`: https://github.com/django/django/blob/main/django/db/models/base.py#L593-L601